ai-lab-demo/README.md

```diff
@@ -0,0 +1,113 @@
+# AI Lab Demo
+
+The purpose of the demo is to demonstrate how to write its first AI App, using Podman AI Lab.
+
+The App we want to bootstrap is a micro-service to search on the Podman Desktop website.
+
+![My first AI App](./my-first-ai-app.png)
+
+## Pre-requisites
+
+- Remove existing `user-catalog.json` file
+- Have a podman machine running using `libkrun`
+- Stop all containers
+- Podman AI Lab version 1.3 or later is installed
+- `GPU support for inference servers` is enabled in AI Lab preferences
+- Pre-download the Mistral model
+- pull the repository `https://github.com/feloy/podman-ai-lab-demo`
```

Comment:
should this repository should be like a subfolder of this ai-lab-demo folder ?
so we have all the materials there

Reply:
I moved the sources to this repo. This will also be used by the blog post (https://github.com/podman-desktop/podman-desktop/pull/9856#discussion_r1856262037)